torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -329,44 +370,65 @@ uint64_t FusionExecutor::computeSharedMemory(
   return total;
 }
 
+uint64_t FusionExecutor::computeSharedMemory(
+    kir::ExpressionEvaluator& expr_eval,
+    const std::vector<const kir::Allocate*>& buffers,
+    const std::vector<glfdc::ExprEvaluator>& memoized,
+    bool align_padding,
+    uint64_t total) {
+  FUSER_PERF_SCOPE("computeSharedMemory::new");
+  size_t i = 0;
+  for (auto smem_alloc : buffers) {
+    if (smem_alloc->alias() == nullptr) {
+      auto mem_inferred = expr_eval.evaluateMemoized(memoized[i]);
+      const auto inferred_val = expr_eval.evaluate(smem_alloc->size());
```

Comment:
Do we still need to look at `smem_alloc->size()`? Can we just use `memoized[i]`?

Reply:
No, its a leftover, I did not remove sorry.

Reply:
Should be fixed